.bazelrc

```diff
@@ -141,8 +141,9 @@ build:gcc --cxxopt=-Wno-missing-requires
 build:gcc --cxxopt=-Wno-dangling-reference
 build:gcc --cxxopt=-Wno-nonnull-compare
 build:gcc --linkopt=-fuse-ld=gold --host_linkopt=-fuse-ld=gold
+build:gcc --host_platform=@envoy//bazel/rbe/toolchains:rbe_linux_gcc_platform
 
-# libc++ default for clang
+# libc++ - default for clang
```

Comment:
Is this needed?

Reply:
i felt it needed a bit of clarifying - without the hyphen it reads as a single sentence which has slightly different connotations